response_operations_ui/views/case.py

```diff
@@ -0,0 +1,45 @@
+from flask import Blueprint, request, render_template, url_for
+from flask_login import login_required
+from werkzeug.utils import redirect
+
+from response_operations_ui.common.mappers import map_ce_response_status
+from response_operations_ui.controllers import case_controller
+from response_operations_ui.forms import ChangeGroupStatusForm
+
+case_bp = Blueprint('case_bp', __name__, static_folder='static', template_folder='templates')
+
+
+@case_bp.route('/<ru_ref>/change-response-status', methods=['GET'])
```

Comment:
Is this the correct url for this? Best to consult with Nic on what he'd like this to be

Reply:
I'll double check but in the prototype it looks this
![image](https://user-images.githubusercontent.com/15050019/36793924-b4688566-1c96-11e8-946f-3eba70107689.png)
I think the query params are just for the prototype as we can look it up